src/Sarif.Multitool/CommandBase.cs

```diff
@@ -12,6 +13,24 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool
 {
     public abstract class CommandBase
     {
+        protected static bool ValidateNonNegativeCommandLineOption<T>(int optionValue, string optionName)
+        {
+            bool valid = true;
+
+            if (optionValue < 0)
+            {
+                string optionDescription = CommandUtilities.GetOptionDescription<T>(optionName);
```

Comment:
>CommandUtilities [](start = 43, length = 16)

I have another PR out that moves this method into `DriverUtilities`, so whoever gets in second will have a small merge conflict to resolve.